ui/src/app/utils/access-utils.tsx

```diff
@@ -364,11 +364,25 @@ export const useIsUserDisabled = () => {
   return disabled;
 };
 
+export const getAccessModuleStatusByNameOrEmpty = (
```

Comment:
these two functions now *always* return an AccessModuleStatus, but it may be initialized to contain only the module name

Reply:
why moduleName and not just an empty object?

Reply:
Ok to have a partial AccessModuleStatus object with all the rest of the time as empty. Do you think we should add a comment for the same since the method name is get status or Empty?

Reply:
Sometimes we access this status alone without an associated name, so we need to keep the name in there to provide context.  Also consumers of this object assume the name is present.  (it should be required in the API but is not unfortunately)

Reply:
update: made it required.

Does this address your comments?  Please chat with with me on Slack if it does not.

Reply:
Yes thanks

Reply:
I need to reveert thee "Required" changee b/c it is causing a variety of problems.  I have another branch working on a refactor which will inculde this change